packages/angular_devkit/build_angular/src/builders/browser/index.ts

```diff
@@ -73,6 +73,12 @@ export type BrowserBuilderOutput = json.JsonObject &
     outputPath: string;
   };
 
+/**
+ * Maximum time for single build/rebuild
+ * This accounts for CI variability.
+ */
+export const BUILD_TIMEOUT = 40000;
```

Comment:
Nit: Add a unit to this variable so future readers know what it represents (I assume milliseconds?). Can add that as a var name suffix or part of the comment.

May also want to consider adding an `_` for readability. I originally read it as `4_000` but it's actually `40_000`.